core/src/odbc_uri.rs

```diff
@@ -610,11 +610,11 @@ mod unit {
             use crate::map;
```

Comment:
You probably want to rename that test to `standard_types` or `complex_types` or something like that.